src/three-components/SmoothControls.ts

```diff
@@ -35,9 +35,9 @@ export interface SmoothControlsOptions {
   // The maximum angle between model-forward and the camera azimuthal position
   maximumAzimuthalAngle?: number;
   // The minimum camera field of view in degrees
-  minimumFov?: number;
+  minimumLogFov?: number;
```

Comment:
`fieldOfView`

Reply:
Separately, its incongruous for us to ask the user to think about "log field of view" here when configuring the controls, but separately set field of view directly in a method like `setFov`. It seems like we could get away with asking the user to set a desired range in degrees and we could `log(fov)` as an internal detail.

Reply:
Good call.